File: pigatize/pigatize.rb
VOWELS = ["a", "e", "i", "o", "u"]

def pigatize(text)
  if starts_with_vowel(text[0])
    pigatized_text = text + "way"
  else
    pigatized_text = text + text.slice!(0) + "ay"
  end
    return pigatized_text
end

def starts_with_vowel(letter)
  return VOWELS.include?(letter)
end

loop do
  print "Please enter a word and I will translate to pig latin: "
  text = gets.chomp
  if text.length == 0
    puts "Please enter a string."
  else
    puts pigatize(text)
  end
end

File: pigatize/pigatize.js
var pigatize = function(text){
    var vowels = /[aeiou]/;
    var word = text[0];
    if(vowels.test(word)){
        return text + "way";
    }else{
        return text.replace(/\b(\w)(\w+)\b/g, '$2$1ay');
    }
};

  var text = prompt("Please enter a word and I will translate to pig latin: ");

  if (text.length === 0) {
    console.log("Please enter a string.");
  }
  else {
    console.log(pigatize(text));
  }


